cmake/modules/VersionInfo.cmake

```
@@ -13,13 +14,18 @@ IF(GIT_FOUND AND NOT FORCE_VERSION)
 		LIST(GET TAG_LIST 0 FORCE_VERSION)
 		STRING(REPLACE "v" "" FORCE_VERSION "${FORCE_VERSION}")
 	ENDIF()
-	IF(TAG_LIST_LENGTH EQUAL 2)
+	IF(TAG_LIST_LENGTH LESS_EQUAL 1)
```

`LESS_EQUAL` is introduced in CMake 3.7, but the current CMake requirement is 3.3.
```suggestion
	IF(TAG_LIST_LENGTH LESS 2)
```

---

@iansannar GitHub won't let me commit @PhysSong's suggesting to your repo.  Can you please click "Commit suggestion"?

---

Fixed via 7c61126.  Also reporting to GitHub support, this seems like an edge-case that collaborators should be able to use.

---

For those interested in the "GitHub won't let me commit" issue above, GitHub finally got back to us... 

> [Tres, correct] ... maintainers should be able to apply suggested changes the same way they can make direct edits to a collaborators fork branch.  It does look like we might have a problem, so I've escalated to our Engineering team to investigate